libs/blocks/global-navigation/utilities/menu/menu.js

```diff
@@ -267,6 +267,28 @@ const decorateColumns = async ({ content, separatorTagName = 'H5' } = {}) => {
   }
 };
 
+const decorateLinksStrip = (content) => {
+  const linksStripEl = content.querySelector('.links-strip');
+  if (!linksStripEl) return;
+  linksStripEl.parentElement.removeChild(linksStripEl);
+  const decorateLink = (el) => {
+    const image = el.querySelector('picture');
+    const link = el.querySelector('a');
+    const imageEl = image ? toFragment`<div class="feds-linksStrip-image">${image}</div>` : '';
+    const contentEl = link ? `<span>${link.textContent}</span>` : '';
+    return toFragment`<div class="feds-linksStrip-item">
+      <a href="${link.href}" class="feds-navLink">
+      ${imageEl}
+      ${contentEl}
+      </a>
+    </div>`;
+  };
+
+  const linksStripWrapper = toFragment`<div class="feds-linksStrip-wrapper"><div class="feds-linksStrip"></div></div>`;
+  linksStripWrapper.querySelector('.feds-linksStrip').append(...[...linksStripEl.children].map(decorateLink));
```

Comment:
We could probably create 2 different fragments and use each for its specific purpose, rather than querying what we create ourselves.